openflow-sdn/openflow-protocol.md

```diff
@@ -0,0 +1,65 @@
+# What is the OpenFlow protocol? 
+
+**Openflow protocol** is a standard in Software Defined Networking(SDN) architecture. This protocol defines the communication between an SDN controller and the network device/agent. Before delving deep into the OpenFlow protocol, here is a background on SDN. 
+
+Before moving further, the term "switch" denotes a network device capable of using OF protocol, and not the Layer 2 device. 
+
+## Background on traditional SDN
```

Comment:
Please add the "Table of Contents" and "Prerequisites" section at the top of your article. It is always a good practice especially since your target readers are beginners. Please refer to this [article](https://www.section.io/engineering-education/deploying-machine-learning-models-using-flask/#table-of-contents) to see an example.